app/reflexes/admin/connected_app_reflex.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Admin
+  class ConnectedAppReflex < ApplicationReflex
+    def create
```

Comment:
I know it's tested in a system spec, but I wonder if we should also have unit test for reflexes ?

Reply:
Good question. I find unit tests good for testing individual methods with all the different code branches. But this method doesn't actually have any branching. And that means that the system spec is providing 100% code coverage. So I wouldn't bother at this point.

Actually, I overlooked one thing. The `authorize!` call is a code branch. We could test permissions in a unit test. Same for `find` and `create!` which could raise errors. But that's well-known behaviour we deal with all the time.